dotnet/src/Connectors/VectorData.Abstractions/VectorData.Abstractions.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.Extensions.VectorData.Abstractions</AssemblyName>
+    <RootNamespace>Microsoft.Extensions.VectorData</RootNamespace>
```

Comment:
Taking into account that assembly name is `Microsoft.Extensions.VectorData.Abstractions`, should the namespace be the same?

Reply:
No, the idea is that you would typically have two packages that contribute to the same namespace, one containing the abstractions and one containing the core code.  The naming convention used by the Microsoft.Extensions packages is where the core package is Microsoft.Extensions.X, while the abstractions for the same namespace is Microsoft.Extensions.X.Abstractions.  This is the same as in SK.Abstractions, where all the abstractions are just in the SK namespace.